ansible_runner/__main__.py

```diff
@@ -217,6 +217,24 @@ def main(sys_args=None):
     parser.add_argument("-a", "--args", dest='module_args',
                         help="Module arguments")
 
+    parser.add_argument("--process-isolation", dest='process_isolation', default=False,
+                        help="Limits what directories on the filesystem the playbook run has access to, defaults to /tmp")
+
+    parser.add_argument("--process-isolation-executable", dest='process_isolation_executable', default="bwrap",
+                        help="Path that an isolated playbook run will use for staging. Defaults to /tmp")
+
+    parser.add_argument("--process-isolation-path", dest='process_isolation_path', default="/tmp",
+                        help="Path that an isolated playbook run will use for staging. Defaults to /tmp")
+
+    parser.add_argument("--process-isolation-hide-paths", dest='process_isolation_hide_paths',
+                        help="List of paths on the system that should be hidden from the playbook run")
+
+    parser.add_argument("--process-isolation-show-paths", dest='process_isolation_show_paths',
+                        help="List of paths on the system that should be exposed to the playbook run")
+
+    parser.add_argument("--process-isolation-venv", dest='process_isolation_venv',
```

Comment:
It looks like this only controls whether a read-only path is added to the bwrap. I think we should switch this to behaving like `show_paths` above and try not to encode any awx-specific logic (not referring to it as `venv`). Instead we can just modify AWX usage to attach it as a `ro` path and adjust the path to the default python executable (which normally comes in through the `--binary` or `RUNNER_BINARY` environment variable)

Reply:
Good point. I will update this.